activities.json

```diff
@@ -142,6 +153,17 @@
     "title": "Feature Policy",
     "url": "https://wicg.github.io/feature-policy"
   },
+  {
+    "ciuName": "imports",
+    "description": "HTML Imports are a way to include and reuse HTML documents in other HTML documents.",
+    "mozBugUrl": null,
+    "mozPosition": "harmful",
```

Comment:
For a "harmful" position I think it's also probably worth saying a little bit more than this about why the alternative solution is better.